anomalib/utils/loggers/comet.py

```diff
@@ -0,0 +1,117 @@
+"""comet logger with add image interface."""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Any, Optional, Union
+
+import numpy as np
+from matplotlib.figure import Figure
+from pytorch_lightning.loggers.comet import CometLogger
+from pytorch_lightning.utilities import rank_zero_only
+
+
+from .base import ImageLoggerBase
+
+
+class AnomalibCometLogger(ImageLoggerBase, CometLogger):
+    """Logger for comet. 
+    
```

Comment:
codacy complains about trailing white space in L18 and L19. Would be good to remove them.